pipelines/_ingredients/ApPipe.yaml

```
@@ -74,6 +75,10 @@ subsets:
       - rbClassify
       - transformDiaSrcCat
       - diaPipe
+  qaMetrics:
+    subset:
+      - analyzeAssocDiaSrcCore
+      - diffimTaskCore
     description: >
       An alias of ApPipe to use in higher-level pipelines.
```

The subset definition got a bit garbled; the description belongs with `apPipe`.
```suggestion
    description: >
      An alias of ApPipe to use in higher-level pipelines.
  qaMetrics:
    subset:
      - analyzeAssocDiaSrcCore
      - diffimTaskCore
```

---

If you could add a description for `qaMetrics`, of course, that would be great.